tjenestebuss-integrasjon/src/main/kotlin/no/nav/pensjon/brev/tjenestebuss/tjenestebussintegrasjon/Routing.kt

```diff
@@ -50,17 +53,19 @@ fun Application.tjenestebussIntegrationApi(config: Config) {
                 println(e)
             }
         }
-        get("/testBestillbrev") {
-            val gc = GregorianCalendar()
-            gc.clear()
-            gc.timeInMillis = 1699538037186L
-
-            val xgc = DatatypeFactory.newInstance().newXMLGregorianCalendar(gc)
+        post("/bestillbrev") {
+            val requestDto = call.receive<BestillBrevDto>()
             try {
-                val brev = arkivTjenestebussService.bestillBrev(ArkivTjenestebussService.BestillBrevDto(xgc))
-                println(brev!!.journalpostId)
+                val arkivResponse = arkivTjenestebussService.bestillBrev(requestDto)
+                val response = """
```

Comment:
Lage en data klasse for respons? `data class BestillBrevResponse(val journalpostId: Long)`